src/components/Header/Header.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import styles from './Header.scss';
+
+const { block, elem } = bem({
+    name: 'Header',
+    classnames: styles
+});
+
+const Header = props => {
+    const { logoSrc, logoLink, logoName, children, ...rest } = props;
+
+    return (
+        <div {...rest} {...block(props)}>
+            <div {...elem('wrapper', props)}>
+                <a href={logoLink}>
+                    <img src={logoSrc} alt={logoName} {...elem('logo', props)} />
+                </a>
+                <div {...elem('menu', props)}>{children}</div>
+            </div>
+        </div>
+    );
+};
+
+Header.displayName = 'Header';
+
+Header.propTypes = {
+    /** path to the logo source file */
+    logoSrc: PropTypes.string.isRequired,
```

Comment:
Maybe prettier to do use 'scoped' props?

```
logo: PropTypes.shape({
   src: PropTypes.string.isRequired,
   link: PropTypes.string,
   title: PropTypes.string
}).isRequired
```

Reply:
it is prettier, but I found a few times that flat structures are easier to handle. For instance, setting defaults on an object is not possible without some magic, since they aren't deep merged. Since these are the only props, I think it is ok to leave them flat.

Reply:
But in this case `src` is required anyway and the others don't necessarily need a default, don't they? But as you say, it's mostly a matter of 'prettier props'